Common/src/main/java/com/tabnineCommon/capabilities/CapabilityNotifier.kt

```diff
@@ -0,0 +1,23 @@
+package com.tabnineCommon.capabilities
+
+import com.intellij.openapi.application.ApplicationManager
+import com.intellij.util.messages.MessageBusConnection
+import com.intellij.util.messages.Topic
+
+fun interface CapabilityNotifier {
+    companion object {
+        private val CAPABILITY_CHANGED_TOPIC = Topic.create("capabilities changed notifier", CapabilityNotifier::class.java)
```

Comment:
same here, the topic name should be `"com.tabnine.capabilitiesChanged"` or something